README.md

```diff
@@ -1,5 +1,15 @@
 # stockpredict [![Build Status](https://travis-ci.org/thedataincubator/stockpredict.svg?branch=master)](https://travis-ci.org/thedataincubator/stockpredict)
 
+Overview:
+This repository contains a stock price predict app that is based on stock market data from Quandl. The goal of this repository is to work as a team to make some changes to the app and get experience on working on production codes. 
+
+Repository Organization:
+static: contains Google stock price file GOOGL_data.txt
```

Comment:
lets change this to static resources in general.

Reply:
will do!